.github/scripts/create-cluster.sh

```diff
@@ -15,4 +15,29 @@ echo "Patching environment '$FLC_ENVIRONMENT' to 'deployed'"
 kubectl patch --namespace "$FLC_NAMESPACE" --type merge --patch '{"spec": {"desiredState": "environment-deployed"}}' flcenvironment "$FLC_ENVIRONMENT"
 
 echo "Waiting up to '$DEFAULT_TIMEOUT' for successful deployment of environment '$FLC_ENVIRONMENT'"
-kubectl wait --namespace "$FLC_NAMESPACE" --timeout="$DEFAULT_TIMEOUT" --for jsonpath='{.status.currentState}'="$DESIRED_STATE" flcenvironment "$FLC_ENVIRONMENT"
+
+# wait until timeout or until the environment is in the desired state
+NEXT_WAIT_TIME=0
+while [[ $NEXT_WAIT_TIME -ne 80 ]]; do
+  # Check if the environment is in the desired state
+  current_state=$(kubectl --namespace "$FLC_NAMESPACE" --for jsonpath='{.status.currentState}' flcenvironment "$FLC_ENVIRONMENT")
+
+  if [[ "$current_state" == "$DESIRED_STATE" ]]; then
```

Comment:
is the additional var her required? The other if branches here use hard-coded strings. I'd suggest to make that consistent.